.changeset/config.json

```diff
@@ -6,5 +6,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@example/**", "cobalt-docs"]
+  "ignore": ["@example/**", "docs"]
```

Comment:
This is actually correct as it was; this scans for the name in `package.json`, not the dir name

Reply:
Ah ok, reverted!